lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -786,7 +786,7 @@ def test_render_html_view_with_preview_mode_when_user_already_has_cert(self):
         # user has already has certificate generated for 'honor' mode
         # so let's try to preview in 'verified' mode.
         response = self.client.get(test_url + '?preview=verified')
-        self.assertNotIn(self.course.display_name, response.content)
+        self.assertNotIn(self.course.display_name.encode('utf-8'), response.content)
```

Comment:
Was it something about the xblock changes that caused this to start coming in as unicode?  This looks like a reasonable change, but I'm not following why this suddenly became a problem.